app/frontend/src/styles/application/jobseekers/subjects-and-grades.scss

```diff
@@ -0,0 +1,57 @@
+#subjects-and-grades {
```

Comment:
far better to use class selectors for css styling. actually far better to avoid ids for this kind of thing in general. what you have done here is pretty specific to this use case (nothing particuarly wrong with that in context of TV application), but i always like or at least try to think when im writing a front end 'components' how can i write this so it can potentially used multiple times on the same page, hence avoiding ids apart from pretty specific reasons

Reply:
even if its unlikely you are going to use it on same page, that mantra is only going to make the code nicer. we'll go to town a bit more on that in pairing we talked about. but id like to see the id here go away tbh